storage/index.md

```diff
@@ -19,10 +19,11 @@ are some downsides:
   kernel. This extra abstraction reduces performance as compared to using
   _data volumes_, which write directly to the host filesystem.
 
-Docker offers three different ways to mount data into a container from the
-Docker host: _volumes_, _bind mounts_, or _`tmpfs` volumes_. When in doubt,
-volumes are almost always the right choice. Keep reading for more information
-about each mechanism for mounting data into containers.
+Docker has two options for containers to store files in the host machine, so
+that the files are persisted even after the container stops: _volumes_, and
+_bind mounts_. If you're running Docker on Linux you can also use a _tmpfs mount_.
+
+Keep reading for more information about these two ways of persisting data.
 
 ## Choose the right type of mount
 
```

Comment:
@joaofnfernandes , down the page (and not part of your edit), can you fix the warning note. It looks malformed. It's part of the **bind mounts** section.